config/routes.rb

```diff
@@ -23,6 +23,7 @@
   resources :repositories, only: [:index, :show, :destroy] do
     post :toggle_star, on: :member
     resources :comments, only: [:create, :destroy]
+    resources :vulnerabilities, only: [:index]
```

Comment:
@mssola we did cherry-pick "Added an authentication layer for the API" commit bcd9c65 from #1299. so some changes go with it. I'll remove them.